### PR TITLE
Update contributors information and alphabetize

### PR DESCRIPTION
**Pull request by btashton**
Signed-off-by: Brennan Ashton <bashton@brennanashton.com>

### CODE REVIEW

**Comment by justinmclean** [comment]
Hi,

The information n this page is not correct. Currently the roster [1] shows.

Mentors are:
Flavio Paiva Junqueira
Junping Du	
Justin Mclean	
Mohammad Asif Siddiqui

PPMC members are:
Abdelatif Guettouche	
Alan Carvalho de Assis	
Alin Jerpelea	
Anthony Merlino	
David Sidrane	
Gregory Nutt	
Masayuki Ishikawa	
Nathan Hartman	
Xiang Xiao

Committers are:
Brennan Ashton

Thanks,
Justin

1. https://whimsy.apache.org/roster/ppmc/nuttx

**Comment by justinmclean** [comment]
Hi,

Note that all Mentors are PPMC members (and IPMC members) and all PPMC members are committers as well.

Thanks,
Justin

**Comment by btashton** [comment]
That is what I looked at to update this.  Are you asking to break it out to also include PPMC or to enumerate all, so for you it would be (Mentor, PPMC, IPMC, Commiiter)

**Comment by justinmclean** [comment]
HI,
> That is what I looked at to update this. Are you asking to break it out to also include PPMC or to enumerate all, so for you it would be (Mentor, PPMC, IPMC, Commiiter)
> 

Most people are listed as committers when they are also PPMC members. It might be useful to break it up that way, a lot of projects do so.

Thanks,
Justin

**Comment by btashton** [comment]
Ok, fully enumerated it is.  Updated the PR.

**Comment by huahang** [comment]
LGTM

File: _data/contributors.yml [hunk 1]
```diff
@@ -18,82 +18,88 @@
...
   org:
 
 - name: David Sidrane
   apacheId: davids5
   githubId: davids5
-  role: Committer
+  role: PPMC, Committer
   org:
 
 - name:	Flavio Paiva Junqueira
```
Comment (xiaoxiang781216) [inline _data/contributors.yml]: remove the extra space?

File: _data/contributors.yml [hunk 2]
```diff
@@ -18,82 +18,88 @@
...
   githubId: acassis
-  role: Committer
+  role: PPMC, Committer
   org:
 
-- name: Abdelatif Guettouche
-  apacheId: aguettouche
-  githubId: Ouss4
-  role: Committer
+- name:	Alin Jerpelea
```
Comment (xiaoxiang781216) [inline _data/contributors.yml]: remove the extra space?

**Pull request merged**